auth/src/main/java/com/redhat/lightblue/rest/auth/ldap/LdapFindUserByUidCommand.java

```diff
@@ -78,7 +84,7 @@ This methods is executed for all types of failure such as run() failure, timeout
      */
     @Override
     protected SearchResult getFallback() {
-        LOGGER.info("Error during the execution of the command. Falling back to the cache");
+        LOGGER.error("Error during the execution of the command. Falling back to the cache");
```

Comment:
Agree. I will switch to warn level . At the first sight I thought this would not impact being this way but as it can have a heavy load it could wast a lot of space and cpu .